velox/common/caching/SsdFile.cpp

```diff
@@ -590,8 +590,11 @@ bool SsdFile::removeFileEntries(
     for (int32_t region : toFree) {
       if (existingWritableRegions.count(region) == 0) {
         writableRegions_.push_back(region);
+        erasedRegionSizes_[region] = 0;
+        regionSizes_[region] = 0;
       }
     }
+    logEviction(toFree);
   }
 
```

Comment:
Shall we also clear suspended_ flag here? And add unit test in this case? Thanks!

Reply:
good point!

Reply:
If suspended_ is cleared, then the ssd space allocation can proceed. If that's the case, do we need to check region pins before we evict them?